crates/networking/p2p/sync.rs

```diff
@@ -181,7 +187,7 @@ async fn download_and_run_blocks(
     store: Store,
 ) -> Result<(), ChainError> {
     loop {
-        let peer = peers.lock().await.get_peer_channels().await;
+        let peer = peers.lock().await.get_peer_channels(Capability::Eth).await;
```

Comment:
You are not dealing with specific versions here? Like eth/68. We can create a issues for this.

Reply:
Created https://github.com/lambdaclass/ethrex/issues/1578